src/main/java/uk/gov/justice/digital/delius/service/ConvictionService.java

```diff
@@ -405,23 +404,19 @@ private LocalDate previouslyKnownTerminationDateOf(Offender offender) {
     }
 
     private Boolean inBreachOf(Offender offender) {
-        if (!probationStatusOf(offender).equals(ProbationStatus.CURRENT)) {
+        if (probationStatusOf(offender) != ProbationStatus.CURRENT) {
             return null;
         }
-        return Optional.of(offender)
```

Comment:
can't see any reason to wrap this in an `Optional` since if `offender` is `null` `Optional.of` will throw a null pointer anyway